src/golang/cmd/server/handler/v2/node_artifact_results_get.go

```diff
@@ -0,0 +1,138 @@
+package v2
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/aqueducthq/aqueduct/cmd/server/handler"
+	"github.com/aqueducthq/aqueduct/lib/database"
+	"github.com/aqueducthq/aqueduct/lib/models"
+	"github.com/aqueducthq/aqueduct/lib/models/shared"
+	"github.com/aqueducthq/aqueduct/lib/repos"
+	"github.com/aqueducthq/aqueduct/lib/response"
+	"github.com/aqueducthq/aqueduct/lib/storage"
+	"github.com/dropbox/godropbox/errors"
+	"github.com/google/uuid"
+)
+
+// This file should map directly to
+// src/ui/common/src/handlers/NodeArtifactResultsGet.tsx
+//
+// Route: /api/v2/workflow/{workflowID}/dag/{dagID}/node/artifact/{nodeID}/results
+// Method: GET
+// Params:
+//	`workflowID`: ID for `workflow` object
+//  `dagID`: ID for `workflow_dag` object
+//	`nodeID`: ID for operator object
+// Request:
+//	Headers:
+//		`api-key`: user's API Key
+// Response:
+//	Body:
+//		`[]response.ArtifactResult`
+
+type NodeArtifactResultsGetHandler struct {
+	nodeGetHandler
+	handler.GetHandler
+
+	Database database.Database
+
+	WorkflowRepo       repos.Workflow
+	DAGRepo            repos.DAG
+	ArtifactRepo       repos.Artifact
+	ArtifactResultRepo repos.ArtifactResult
+}
+
+func (*NodeArtifactResultsGetHandler) Name() string {
+	return "NodeArtifactResultsGet"
+}
+
+func (h *NodeArtifactResultsGetHandler) Prepare(r *http.Request) (interface{}, int, error) {
+	return h.nodeGetHandler.Prepare(r)
+}
+
+func (h *NodeArtifactResultsGetHandler) Perform(ctx context.Context, interfaceArgs interface{}) (interface{}, int, error) {
+	args := interfaceArgs.(*nodeGetArgs)
```

Comment:
Not much need to review here. This is copied from current `list_artifact_results` route.